easybuild/easyconfigs/c/ccache/ccache-4.6.3-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,47 @@
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# Updated to 4.6.1: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'ccache'
+version = '4.6.3'
+
+homepage = 'https://ccache.dev/'
+description = """Ccache (or “ccache”) is a compiler cache. It speeds up recompilation by
+caching previous compilations and detecting when the same compilation is being done again"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/ccache/ccache/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['f46ba3706ad80c30d4d5874dee2bf9227a7fcd0ccaac31b51919a3053d84bd05']
+
+osdependencies = [('glibc-static', 'libc6-dev', 'glibc-devel')]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('Ninja', '1.10.2'),
+    ('zstd', '1.5.2', ),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('hiredis', '1.0.2'),
+]
+
+# use BFD linker rather than default ld.gold (required on CentOS 8)
+preconfigopts = 'LDFLAGS=" -fuse-ld=bfd"'
+configopts = '-DENABLE_DOCUMENTATION=OFF -DENABLE_IPO=ON -DZSTD_LIBRARY="$EBROOTZSTD/lib/libzstd.a" '
+# disable hunt for faster linker, since using ld.gold may fail (on CentOS 8, for example)
+configopts += '-DUSE_FASTER_LINKER=OFF'
```

Comment:
there are some other configopts in ccache-4.6.3.eb that are missing here.
please add them back (or explain why they are no longer needed).